src/Chapter1.hs

```diff
@@ -620,8 +625,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Integral a => a -> a
+sumLast2 n = lastD + secondToLastD
+  where
+    n' = abs n
+    lastD = n' `mod` 10
+    secondToLastD = (div n' 10) `mod` 10
```

Comment:
Nice solution! 🍬 

You could also reuse `lastDigit` from the previous task instead of `mod 10` 🙂